src/index.js

```diff
@@ -1,34 +1,31 @@
 import 'bootstrap/dist/css/bootstrap.min.css';
-import './reset.css'
+import './reset.css';
 import './style.css';
-import * as Update from './Update'
-import Onclic from './onclic'
+import * as Update from './Update';
+import Onclic from './onclic';
 import * as helper from './style_helpers';
 
-const apikey ='e07023ef36e4fb48fc91efb895731682';
-helper.btn('btnFind', ()=>{
-  const cityname = document.getElementById('city').value;
-  const link = 'https://api.openweathermap.org/data/2.5/forecast?q='+cityname+'&appid='+apikey;
-  getData(link);
-}, 'empty');
-
-
-async function getData(nurl){
-  let response = await fetch(nurl ,{mode:"cors"});
+async function getData(nurl) {
```

Comment:
- [ ] [REQUIRED] This functions should be using an arrow function instead of the `function` keyword
![image](https://user-images.githubusercontent.com/73254040/101058824-3274b880-3553-11eb-90e6-764395daa45b.png)
[Source](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)